geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/executor/sortedset/ZAddExecutor.java

```diff
@@ -47,7 +48,7 @@ public RedisResponse executeCommand(Command command, ExecutionHandlerContext con
 
     return RedisResponse
         .integer(redisSortedSetCommands.zadd(command.getKey(),
-            commandElements.subList(optionsFoundCount + 2, commandElements.size()),
+            new ArrayList<>(commandElements.subList(optionsFoundCount + 2, commandElements.size())),
```

Comment:
Hmm. Do you remember which test broke? I reverted this change and ran the new tests in this PR without any problems. Sorry to hold this up but I'd hate for us to add an unnecessary allocation in a potentially fairly hot path.

Reply:
Looking at [the Concourse job](https://concourse.apachegeode-ci.info/builds/41851), it seems like `shouldDistributeDataAmongCluster_givenConcurrentlyAddingDifferentSets()` failed due to `java.io.NotSerializableException: java.util.ArrayList$SubList`

Reply:
Yup, and running locally, sometimes the other ZAddDUnitTest test would fail as well.

Reply:
Yes, you're right. I must have hit the one time it actually all passed. I see it failing now. Sorry for the noise.